core/domain/dependency_registry.py

```diff
@@ -46,18 +46,11 @@ def get_angular_modules(cls, dependency_id):
             dependency_id, [])
 
     @classmethod
-    def get_deps_html_and_angular_modules(cls, dependency_ids):
+    def get_deps_html(cls, dependency_ids):
         """Returns data needed to load the given dependencies.
 
-        The return value is a 2-tuple. The first element of the tuple is the
-        additional HTML to insert on the page. The second element of the tuple
-        is a de-duplicated list of strings, each representing an additional
-        angular module that should be loaded.
+        The return value is the additional HTML to insert on the page.
```

Comment:
It would be great if you can add the Args and Returns in the docstring*

Reply:
The dependency system has been removed so this file no longer exists.